src/file/FileController.ts

```diff
@@ -68,7 +68,8 @@ export abstract class FileController {
 				throw new ConnectionError("currently offline")
 			}
 		} finally {
-			// we don't necessarily know when the user is done with the temporary file that was opened.
+			// we don't necessarily know when the user is done with the temporary file that was opened
+			// because we don't have a handle to the opening application/process.
```

Comment:
but this does mean that we might nuke the opened file if we try to download another one

Reply:
previously, that was handled by non clobbering file names. now, downloaded attachments inside the temp folder always have random names, so it should be okay